account_factoring_receivable_balance/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# © 2024 Open Source Integrators, Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    factor_journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        domain="[('factor_type', '!=', False)]",
```

Comment:
Required in multi-company environment
```suggestion
        domain="[('factor_type', '!=', False)]",
        company_dependent=True,
```